spec/lib/student_greeting_checker_spec.rb

```diff
@@ -58,6 +58,16 @@
           expect(checker.send(:talk_page_blank?)).to eq(true)
         end
       end
+
+      describe '#update_greeting_status' do
+        it 'should not try and pluck username from greeters' do
```

Comment:
This is not a useful test, as it is about a specific implementation rather than the expected behavior of the system.